test/OrchardCore.Tests.Functional/cypress-commands/src/tenants.js

```diff
@@ -1,9 +1,9 @@
 Cypress.Commands.add("login", function({ prefix = ""}={}) {
   const config = Cypress.config('orchard');
   cy.visit(`${prefix}/login`);
-  cy.get("#UserName").type(config.username);
-  cy.get("#Password").type(config.password);
-  cy.get("#UserName").closest('form').submit();
+  cy.get("#LoginForm_UserName").type(config.username);
+  cy.get("#LoginForm_Password").type(config.password);
+  cy.get("#LoginForm_UserName").closest('form').submit();
```

Comment:
Humm, how did this pass in the past?

Reply:
The "dist" folder was fine, it was changed manually. And the "dist" is generated. So as long as we didn't generate a new "dist" it was working. Kind of the same problem we had with the static assets, so we could add this to the validation action such that the source and generated scripts are never out of sync.